src/dowel/csv_output.py

```diff
@@ -41,13 +42,25 @@ def record(self, data, prefix=''):
                     extrasaction='ignore')
                 self._writer.writeheader()
 
-            if to_csv.keys() != self._fieldnames:
-                self._warn('Inconsistent TabularInput keys detected. '
```

Comment:
I think this means that the `_warn` method itself can also be deleted, right?

Reply:
Yes. But what if there are situations where we should warn? If so, `_warn` should be kept and updated.
For the purpose of this issue, it can be deleted.